addons/website_forum/static/src/js/website_forum.js

```diff
@@ -118,35 +118,71 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
             var editorKarma = $textarea.data('karma') || 0; // default value for backward compatibility
             var hasFullEdit = parseInt($("#karma").val()) >= editorKarma;
             var toolbar = [
-                ['style', ['style']],
-                ['font', ['bold', 'italic', 'underline', 'clear']],
-                ['para', ['ul', 'ol', 'paragraph']],
-                ['table', ['table']],
+                [
+                    [
+                        'ParagraphButton',
+                        'Heading1Button',
+                        'Heading2Button',
+                        'Heading3Button',
+                        'Heading4Button',
+                        'Heading5Button',
+                        'Heading6Button',
+                        'PreButton',
+                    ],
+                ],
+                [
+                    'BoldButton',
+                    'ItalicButton',
+                    'UnderlineButton',
+                    'RemoveFormatButton'
+                ],
+                [
+                    'OrderedListButton',
+                    'UnorderedListButton',
+                    [
+                        'AlignLeftButton',
+                        'AlignCenterButton',
+                        'AlignRightButton',
+                        'AlignJustifyButton',
+                    ],
+                ],
+                ['TableButton'],
             ];
             if (hasFullEdit) {
-                toolbar.push(['insert', ['link', 'picture']]);
+                toolbar.push([
+                    'OdooLinkButton',
+                    'UnlinkButton',
+                    'OdooMediaButton',
+                ]);
             }
-            toolbar.push(['history', ['undo', 'redo']]);
-
+            toolbar.push(['UndoButton', 'RedoButton']);
             var options = {
-                height: 200,
-                minHeight: 80,
-                toolbar: toolbar,
-                styleWithSpan: false,
-                wrapperClass: 'note-editable flex-grow-1 ',
+                // toolbarLayout: toolbar,
+                height: '100%',
+                wrapperClass: 'note-editable o_editable flex-grow-1 ',
                 recordInfo: {
                     context: self._getContext(),
                     res_model: 'forum.post',
                     res_id: +window.location.pathname.split('-').pop(),
                 },
+                enableResizer: true,
                 value: textarea.value.trim() ? textarea.value : '<p><br/></p>',
+                location: [$textarea.next()[0], 'replace'],
```

Comment:
This is a hack (do not merge with this!): it replaces the spinner with the editor... Problem is, there doesn't seem to be a system to remove the spinner otherwise so we need to discuss it.

Reply:
cc @Goaman

Reply:
The one who create the spinner should remove the spinner. Who create it?

Reply:
`addons/web_editor/static/src/js/frontend/loader_loading.js`
```js
document.addEventListener('DOMContentLoaded', () => {
    // Standard loop for better browser support
    var textareaEls = document.querySelectorAll('textarea.o_wysiwyg_loader');
    for (var i = 0; i < textareaEls.length; i++) {
        var textarea = textareaEls[i];
        var wrapper = document.createElement('div');
        wrapper.classList.add('position-relative', 'o_wysiwyg_wrapper');

        var loadingElement = document.createElement('div');
        loadingElement.classList.add('o_wysiwyg_loading');
        var loadingIcon = document.createElement('i');
        loadingIcon.classList.add('text-600', 'text-center',
            'fa', 'fa-circle-o-notch', 'fa-spin', 'fa-2x');
        loadingElement.appendChild(loadingIcon);
        wrapper.appendChild(loadingElement);

        textarea.parentNode.insertBefore(wrapper, textarea);
        wrapper.insertBefore(textarea, loadingElement);
    }
});
```
I don't see any code in there removing it so I'm a bit confused... :shrug:

Reply:
https://github.com/odoo/odoo/pull/59828/files#diff-78a0f213dce82e60024b1e9f781d439246a6fc696a044741fb6708f88c637839L8

Reply:
Thanks now it's clear.

Reply:
Thanks now it's clear.